hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rollout/AddUpdateRolloutWindowLayout.java

```diff
@@ -1106,18 +1106,17 @@ private void setAutoStartType(final Rollout rollout) {
 
     private enum ERROR_THRESHOLD_OPTIONS {
 
-        PERCENT("%"),
+        PERCENT("label.errorthreshold.option.percent"),
+        COUNT("label.errorthreshold.option.count");
 
-        COUNT("Count");
+        private String value;
 
-        private final String value;
-
-        ERROR_THRESHOLD_OPTIONS(final String val) {
-            value = val;
+        ERROR_THRESHOLD_OPTIONS(final String value) {
```

Comment:
Constructor can be private as well

Reply:
IMHO this is redundant, since it will be private anyway, right?